aws/cf.go

```diff
@@ -107,12 +122,15 @@ func (o stackOutput) dnsName() string {
 	return o[outputLoadBalancerDNSName]
 }
 
+func (o stackOutput) lbARN() string {
+	return o[outputLoadBalancerARN]
+}
+
 func (o stackOutput) targetGroupARNs() (arns []string) {
-	if arn, ok := o[outputTargetGroupARN]; ok {
-		arns = append(arns, arn)
-	}
-	if arn, ok := o[outputHTTPTargetGroupARN]; ok {
-		arns = append(arns, arn)
+	for k, v := range o {
```

Comment:
hmm, I think it's better to have the 2 map lookups instead of a range loop over the whole map. 
Do we have more than these 2 map lookups, now?